src/comm_manager.cpp

```diff
@@ -403,6 +412,49 @@ void CommManager::heartbeat_callback(void)
   this->send_heartbeat();
 }
 
+void CommManager::config_set_callback(uint8_t device, uint8_t configuration)
+{
+  uint8_t requested_device{device};
+  if(device >=Configuration::DEVICE_COUNT)
+    device = Configuration::DEVICE_COUNT;
+  ConfigManager::config_response resp = RF_.config_manager_.attempt_set_configuration(static_cast<device_t>(device), configuration);
+  comm_link_.send_config_status(sysid_, requested_device, resp.successful, resp.reboot_required, resp.message);
+}
+
+void CommManager::config_request_callback(uint8_t device)
+{
+  if(device < Configuration::DEVICE_COUNT)
+    send_config_value(static_cast<device_t>(device));
+  if(device == 0xff)
```

Comment:
This works... I'm not a huge fan of magic numbers like this though. I'd probably prefer to follow the parameters model and have a `config_request_list` message type

Reply:
I now use a rosflight command message to request all configuration and device information.